OBJECT/SetPropTintIndex.md

```diff
@@ -0,0 +1,23 @@
+---
+ns: OBJECT
+aliases: ["0x31574B1B41268673"]
+---
+## SET_PROP_TINT_INDEX
+
+```c
+// 0x31574B1B41268673
+void SET_PROP_TINT_INDEX(cs_type(Any) Object object, cs_type(Any) int tint);
```

Comment:
`Vehicle`, `Object` and other related types [here](https://github.com/citizenfx/fivem/blob/030778a167242f79e0f59cd8d9c04b970e3b75c2/ext/natives/codegen_types.lua#L16-L37) map to `int` so using `cs_type` is generally not required for those. Also not required for `Any` to `int`.